cli/commands/logs/index.ts

```diff
@@ -0,0 +1,76 @@
+import { AwilixContainer } from 'awilix';
+import { CommandHandler } from "../..";
+import { assertExists, assertIsISOString, isValidTimeRange } from "../../utils";
+import { GetAgentLogs } from '../../utils/get.agent.logs';
+
+const SECONDS_IN_A_DAY = 86400;
+
+export default function provideLogs(
+  container: AwilixContainer
+): CommandHandler {
+  assertExists(container, 'container')
+
+  return async function logs(cliArgs: any) {
+    const getAgentLogs = container.resolve<GetAgentLogs>('getAgentLogs')
+    
+    const agentId = cliArgs.agentId
```

Comment:
If we wanted some sanity check, we could check 'starts with 0x' or something - but I'd like to limit the number of places that make assumptions about the structure there.  (in case we change it)